go/test/endtoend/sharding/base_sharding.go

```diff
@@ -271,6 +281,30 @@ func InsertToTablet(t *testing.T, query string, vttablet cluster.Vttablet, ks st
 	_, _ = vttablet.VttabletProcess.QueryTablet("commit", ks, true)
 }
 
+// InsertMultiValueToTablet inserts a multiple values to vttablet
+func InsertMultiValueToTablet(t *testing.T, tablet cluster.Vttablet, keyspaceName string, tableName string,
```

Comment:
this can be renamed to `InsertMultiValues`. I don't think it is important to highlight in the function name that we are doing it directly on tablet vs going through vtgate.